AY2021S2-CS2103-T14-1/tp
# Docs(DG): Update model component of DG

## What this does
This PR:
1. Closes #355 by updating the Model sub-section of the DG
2. Closes #191 

## How to test
1. cd to `docs/` folder of the project.
2. Run `bundle exec jekyll serve`.
3. Open the jekyll-hosted website in your browser (default address: http://127.0.0.1:4000).
4. Check that the DG has the changes listed above
    - Ensure links are pointing to the right place
    - Ensure that diagrams are accurate
    - Ensure that descriptions of each class are accurate
